primitives/transaction-pool/src/lib.rs

```diff
@@ -21,13 +21,6 @@
 #![cfg_attr(not(feature = "std"), no_std)]
 
 pub mod runtime_api;
-#[cfg(feature = "std")]
-pub mod error;
-#[cfg(feature = "std")]
-mod pool;
-
-#[cfg(feature = "std")]
-pub use pool::*;
 
 pub use sp_runtime::transaction_validity::{
```

Comment:
I guess it's not really needed that much any more?

Reply:
I think as the primitives crate has got 'cleaner' a lot of these depednencies have moved up into the sc-transaction-pool crate.